src/test/java/com/blackduck/integration/polaris/common/configuration/PolarisServerConfigBuilderTest.java

```diff
@@ -113,7 +113,7 @@ public void testResolveAccessTokenFromPolarisHomeEnvVar() throws IOException {
         Assertions.assertFalse(polarisServerConfigBuilder.isValid());
 
         polarisServerConfigBuilder.setProperties(properties.entrySet());
-        assertTrue(polarisServerConfigBuilder.isValid());
+        Assertions.assertTrue(polarisServerConfigBuilder.isValid());
```

Comment:
I see that we are importing all of the below:

```
import static junit.framework.TestCase.assertNotNull;
import static junit.framework.TestCase.assertTrue;
import org.junit.jupiter.api.Assertions;
```

You think it Weill be better to just use `assertTrue` as we are already importing it? Or I'm fine to use `Assertions` but then it might be better to update other usages of `assertTrue` to avoid multiple imports of the same package in different ways. Valid wherever applicable throughout the PR.

Reply:
updated accordingly